controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
+ * @route GET /v1/transparency
+ * @access PUBLIC
+ */
+exports.getTotalAmount = catchAsyncError(async (req, res) => {
+  const startDate = new Date(req.query.startDate);
+  const endDate = new Date(req.query.endDate);
+
+  const totalDonations = await Donation.getTotal(startDate, endDate);
```

Comment:
I would believe we want to wrap this call in a `try... catch` block so we can properly handle the failure state.

So something like:

```
let totalDonations = [];

try {
    totalDonations = await Donation.getTotal(startDate, endDate);
    return res.status(200).json({
       status: true,
       data: totalDonations
    });
}
catch (error) {
   return res.status(500).json({
      status: false,
      data: [],
     error
  });
}